home/utils.py

```diff
@@ -1,16 +1,47 @@
 import csv
 import io
-import sys
+from datetime import datetime
+from typing import List, Tuple
 
+from openpyxl.styles import Border, Color, Font, NamedStyle, PatternFill, Side
+from openpyxl.workbook import Workbook
+from openpyxl.worksheet.worksheet import Worksheet
 from taggit.models import Tag
 from wagtail import blocks
+from wagtail.documents.models import Document
 from wagtail.images.blocks import ImageChooserBlock
 from wagtail.images.models import Image
 from wagtail.models import Locale
+from wagtail.query import PageQuerySet
 from wagtail.rich_text import RichText
+from wagtailmedia.models import Media
 
 from home.models import ContentPage, HomePage
 
+fieldnames = [
+    "parent",
+    "web_title",
+    "web_subtitle",
+    "web_body",
+    "whatsapp_title",
+    "whatsapp_body",
+    "messenger_title",
+    "messenger_body",
+    "viber_title",
+    "viber_body",
+    "translation_tag",
+    "tags",
+    "quick_replies",
+    "triggers",
+    "locale",
+    "next_prompt",
+    "image_link",
+    "doc_link",
+    "media_link",
+    "related_pages",
+]
+export_filename = f'exported_content_{datetime.now().strftime("%Y%m%d")}'
```

Comment:
The convention for these kind of global variables is screaming snake case, ie. `EXPORT_FILENAME`